code/modules/atmospherics/machinery/other/meter.dm

```diff
@@ -3,7 +3,7 @@
 	desc = "It measures something."
 	icon = 'icons/obj/atmospherics/pipes/meter.dmi'
 	icon_state = "meterX"
-	layer = GAS_PUMP_LAYER
+	layer = 2.55
```

Comment:
Need to be in relation to a define, did you mean to make this number just over the gas pump layer? If so this sort of thing is decent
```suggestion
	layer = GAS_PUMP_LAYER + 0.01
```
Otherwise you probably want a new define in here
https://github.com/tgstation/tgstation/blob/f4a280c4b67218b619c7a256a214fe0f8b4c1f61/code/__DEFINES/layers.dm#L50-L55

Reply:
added a define